uber/templates/registration/form.html

```diff
@@ -242,6 +242,20 @@
     </div>
 </div>
 
+{% if attendee.promo_code_groups or attendee.promo_code and attendee.promo_code.group %}
+<div class="form-group">
+    <label class="col-sm-3 control-label">Promo Code Group</label>
+    <div class="col-sm-6 form-control-static">
+      {% for group in attendee.promo_code_groups %}
+        This attendee is the buyer/group leader for <strong><a href="../groups/promo_code_group_form?id={{ group.id }}" target="_blank">{{ group.name }}</a></strong>.
```

Comment:
This new display kind of breaks tradition in that it's structured like an English sentence that tells you all the relevant information regarding the PromoCodeGroup relationship. I feel like that's more intuitive and readable, but maybe it's just wordy.

Reply:
Seems fine to me.  I'm not a UI/UX guy (obviously!) but this seems like it'll be clear to any admins reading it.